fisco-bcos/main/main.cpp

```diff
@@ -124,6 +124,7 @@ int main(int argc, const char* argv[])
 {
     /// set LC_ALL
     setDefaultOrCLocale();
+    std::set_terminate([]() { std::cerr << "terminate handler called" << endl; });
```

Comment:
This handler will only print the error message, and not calls `abort` or `exit`, will the program still runs when the unexpected exception happened.

Reply:
yes, you are right, call abort().